### ISSUE TITLE
airbnb eslint style guidelines and padded-blocks with ES6

### ISSUE THREAD

**Issue by justin808** [eslint/eslint#2387]
A related issue if using the airbnb eslint style guidelines:

https://github.com/justin808/react-rails-tutorial/

run 'bin/lint'

The error below is set to Warn. There should be no error.

```
29:17  error    Block must not be padded by blank lines  padded-blocks
```

```javascript
import React from 'react';
import Comment from './Comment';

const CommentList = React.createClass({
  displayName: 'CommentList',

  propTypes: {
    comments: React.PropTypes.array
  },

  render() {
    const reversedData = this.props.comments.slice(0).reverse();
    const commentNodes = reversedData.map((comment, index) => {
      // `key` is a React-specific concept and is not mandatory for the
      // purpose of this tutorial. if you're curious, see more here:
      // http://facebook.github.io/react/docs/multiple-components.html#dynamic-children
      return (
        <Comment author={comment.author} key={index} text={comment.text}/>
      );
    });

    return (
      <div className="commentList">
        {commentNodes}
      </div>
    );
  }
});

export default CommentList;
```

**Comment by lo1tuma** [comment]
The airbnb eslint styleguide uses babel-eslint, so I would guess you are running into the same problem as reported there https://github.com/babel/babel-eslint/issues/33.

Can you try linting your example with the default parser espree?

**Comment by justin808** [comment]
@lo1tuma I'm using babel (es6) syntax. Would espree work the same?

**Comment by nzakas** [comment]
@justin808 see http://eslint.org/docs/user-guide/configuring for information on language support.

**Comment by justin808** [comment]
With esprima, JSX files are not parsed by default.

Maybe that's the issue. Is there a way to tell eslint to grab files names *.jsx?

```
npm i -g esprima                                                                                                                                                   1 ↵ ✹ [6:31:25]
/usr/local/bin/esparse -> /usr/local/lib/node_modules/esprima/bin/esparse.js
/usr/local/bin/esvalidate -> /usr/local/lib/node_modules/esprima/bin/esvalidate.js
esprima@2.2.0 /usr/local/lib/node_modules/esprima
➜  justin@JustinMbp-2:~/clients/dashboard/client (feature/slim-lint u=) ✗ eslint .                                                                                                                                                               ✹ [6:32:09]

server.js
  26:0  error  Expected space or tab after // in comment  spaced-line-comment
  72:2  error  Expected space or tab after // in comment  spaced-line-comment
  93:2  error  Expected space or tab after // in comment  spaced-line-comment

webpack.hot.config.js
  28:4  error  Expected space or tab after // in comment  spaced-line-comment
  34:4  error  Expected space or tab after // in comment  spaced-line-comment

✖ 5 problems (5 errors, 0 warnings)
```

**Comment by nzakas** [comment]
We don't use Esprima by default, we use Espree. Please see the link in my previous comment for how to configure ESLint for language options. You can also pass JSX files directly on the command line or use `--ext jsx` (see http://eslint.org/docs/user-guide/command-line-interface)

**Comment by justin808** [comment]
@nzakas:

The fix for my problem was to run the command like:
```
eslint . --ext .jsx
```

1. **Need to put a period with the jsx**
2. Any way to make .jsx a default in the `.eslintrc` file?

**Comment by nzakas** [comment]
There's no way to make file extension changes in eslintrc

**Comment by nzakas** [comment]
It looks like there's nothing else to fix here, am I correct?

**Comment by gyandeeps** [comment]
I am going to close this as I think the initial issue was resolved.
Feel free to reopen if it needs more discussion.
Thanks
